src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,31 @@
+@Artiom
+Feature: Wish List
+
+  Background:
+    Given user is on landing page
+
+  Scenario: Create Wish List
+    Given user is logged in
+    And user is on MY WISHLISTS page
+    And user type on name field My favourite
+    When user click on Save button
+    Then Wishlist is created
```

Comment:
Here it will be better to specify which Wishlist is created, otherwise you will need to save Whishlist name in ScenarioContext, it's up to you